docs/api.rst

```diff
@@ -144,6 +145,105 @@ Variable recombination rates
 .. autoclass:: msprime.RecombinationMap
     :members:
 
+.. _sec_api_simulation_models:
+
+*****************
+Simulation models
+*****************
+
+The default simulation model in ``msprime`` is the standard coalescent with recombination
+model. We also support a number of different models, which are documented in this section.
+
+Simulations models are specified using the ``model`` parameter to
+:func:`.simulate`. This parameter can either take the form of a
+string describing the model (e.g. ``model="smc"``) or an instance of a
+model definition class (e.g ``model=msprime.SmcApproxCoalescent(1)``).
+The available models are documented in the following subsections.
```

Comment:
maybe don't use `smc` as the example, since it's nice to have but it's not really a popluation model and we don't want to encourage people to use it (unless they know what they are doing) - `dtwf`?